pkg/kapp/clusterapply/waiting_changes.go

```diff
@@ -119,7 +119,11 @@ func (c *WaitingChanges) WaitForAny() ([]WaitingChange, error) {
 		}
 
 		if time.Now().Sub(startTime) > c.opts.Timeout {
-			return nil, fmt.Errorf("Timed out waiting after %s", c.opts.Timeout)
+			trackedResourcesDesc := ""
+			for _, change := range c.trackedChanges {
+				trackedResourcesDesc = trackedResourcesDesc + change.Graph.Change.Resource().Description() + "\n "
+			}
+			return nil, fmt.Errorf("Timed out waiting after %s for resources:\n %+v", c.opts.Timeout, trackedResourcesDesc)
```

Comment:
Should we be using `%s` instead of `%+v`?
`%+v` seems to be like something we would use to print structs with keys 🤔

Reply:
+1

Reply:
Yes %+v is used to print struct value. I was using this to check struct info and forgot to change later. Updated now